addons/website_forum/data/forum_default_faq.xml

```diff
@@ -46,33 +46,33 @@
+                                            <li><span style="font-weight: bold;">Answers should not add or expand questions</span>. Instead either edit the question or add a question comment.</li>
+                                            <li><span style="font-weight: bold;">Answers should not comment other answers</span>. Instead add a comment on the other answers.</li>
+                                            <li><span style="font-weight: bold;">Answers shouldn't just point to other Questions</span>. Instead add a question comment indication "Possible duplicate of...". However, it's ok to include links to other questions or answers providing relevant additional information.</li>
+                                            <li><span style="font-weight: bold;">Answers shouldn't just provide a link a solution</span>. Instead provide the solution description text in your answer, even if it's just a copy/paste. Links are welcome, but should be complementary to answer, referring sources or additional reading.</li>
                                         </ul>
                                     </div>
                                 </div>
                             </div>
                             <div class="card bg-white" data-name="Item">
                                 <a href="#" role="tab" data-toggle="collapse" aria-expanded="false" class="collapsed card-header" data-target="#collapse3">
-                                    <b>What should I avoid in my answers?</b>
+                                    <span style="font-weight: bold;">What should I avoid in my answers?</span>
                                 </a>
                                 <div id="collapse3" class="collapse"  data-parent="#myCollapse" role="tabpanel">
                                     <div class="card-body">
-                                        <p><b>Answers should not add or expand questions</b>. Insteadeither edit the question or add a comment.</p>
-                                        <p><b>Answers should not comment other answers</b>. Instead add a comment on the other answers.</p>
-                                        <p><b>Answers shouldn't just point to other questions</b>.Instead add a comment indicating <i>"Possible duplicate of..."</i>. However, it's fine to include links to other questions or answers providing relevant additional information.</p>
-                                        <p> <b>Answers shouldn't just provide a link a solution</b>. Instead provide the solution description text in your answer, even if it's just a copy/paste. Links are welcome, but should be complementary to answer, referring sources or additional reading.</p>
```

Comment:
:grimacing: 

Did you replace all the `<b>...</b>` one by one by hand? :open_mouth: A search and replace would not create that type of error and would be done a lot more quickly, no? :grimacing: Can you check the rest of the commit again? I saw that one fast-scrolling the PR, so a careful review may reveal more mistakes :grimacing:

Reply:
I wanted to replace the <b> tag only in the editable areas.. so it is possible that I made mistakes at this level. I will double check